src/Options/Definitions.js

```diff
@@ -3,604 +3,626 @@
+  },
+  databaseOptions: {
+    env: 'PARSE_SERVER_DATABASE_OPTIONS',
+    help: 'Options to pass to the mongodb client',
+    action: parsers.objectParser,
+  },
+  databaseURI: {
+    env: 'PARSE_SERVER_DATABASE_URI',
+    help: 'The full URI to your database. Supported databases are mongodb or postgres.',
+    required: true,
+    default: 'mongodb://localhost:27017/parse',
+  },
+  directAccess: {
+    env: 'PARSE_SERVER_ENABLE_EXPERIMENTAL_DIRECT_ACCESS',
+    help:
+      'Replace HTTP Interface when using JS SDK in current node runtime, defaults to false. Caution, this is an experimental feature that may not be appropriate for production.',
+    action: parsers.booleanParser,
+    default: false,
+  },
+  dotNetKey: {
+    env: 'PARSE_SERVER_DOT_NET_KEY',
+    help: 'Key for Unity and .Net SDK',
+  },
+  emailAdapter: {
+    env: 'PARSE_SERVER_EMAIL_ADAPTER',
+    help: 'Adapter module for email sending',
+    action: parsers.moduleOrObjectParser,
+  },
+  emailVerifyTokenReuseIfValid: {
+    env: 'PARSE_SERVER_EMAIL_VERIFY_TOKEN_REUSE_IF_VALID',
+    help:
+      'an existing email verify token should be reused when resend verification email is requested',
+    action: parsers.booleanParser,
+    default: false,
+  },
+  emailVerifyTokenValidityDuration: {
+    env: 'PARSE_SERVER_EMAIL_VERIFY_TOKEN_VALIDITY_DURATION',
+    help: 'Email verification token validity duration, in seconds',
+    action: parsers.numberParser('emailVerifyTokenValidityDuration'),
+  },
+  enableAnonymousUsers: {
+    env: 'PARSE_SERVER_ENABLE_ANON_USERS',
+    help: 'Enable (or disable) anonymous users, defaults to true',
+    action: parsers.booleanParser,
+    default: true,
+  },
+  enableExpressErrorHandler: {
+    env: 'PARSE_SERVER_ENABLE_EXPRESS_ERROR_HANDLER',
+    help: 'Enables the default express error handler for all errors',
+    action: parsers.booleanParser,
+    default: false,
+  },
+  enableSingleSchemaCache: {
+    env: 'PARSE_SERVER_ENABLE_SINGLE_SCHEMA_CACHE',
+    help:
+      'Use a single schema cache shared across requests. Reduces number of queries made to _SCHEMA, defaults to false, i.e. unique schema cache per request.',
+    action: parsers.booleanParser,
+    default: false,
+  },
+  encryptionKey: {
+    env: 'PARSE_SERVER_ENCRYPTION_KEY',
+    help: 'Key for encrypting your files',
+  },
+  expireInactiveSessions: {
+    env: 'PARSE_SERVER_EXPIRE_INACTIVE_SESSIONS',
+    help: 'Sets wether we should expire the inactive sessions, defaults to true',
+    action: parsers.booleanParser,
+    default: true,
+  },
+  fileKey: {
+    env: 'PARSE_SERVER_FILE_KEY',
+    help: 'Key for your files',
+  },
+  filesAdapter: {
+    env: 'PARSE_SERVER_FILES_ADAPTER',
+    help: 'Adapter module for the files sub-system',
+    action: parsers.moduleOrObjectParser,
+  },
+  fileUpload: {
+    env: 'PARSE_SERVER_FILE_UPLOAD_OPTIONS',
+    help: 'Options for file uploads',
+    action: parsers.objectParser,
+    default: {},
```

Comment:
Any way to remove the default empty object ? An empty object can be translated to undefined.

Reply:
Why do you want to remove it?

Reply:
@Moumouls Do you want to elaborate your concern? Otherwise I’ll merge.